control-plane/agents/src/common/errors.rs

```diff
@@ -170,6 +170,8 @@ pub enum SvcError {
     Internal { details: String },
     #[snafu(display("Invalid Arguments"))]
     InvalidArguments {},
+    #[snafu(display("Invalid Label: {} ", labels))]
+    InvalidLabel { labels: String },
```

Comment:
We can add a resource kind here.

```suggestion
    #[snafu(display("Invalid {}, labels: {} ", resource_kind, labels))]
    InvalidLabel { labels: String, resource_kind: ResourceKind },
```